package.json

```diff
@@ -1,9 +1,18 @@
 {
   "name": "orbit-browser-extension",
   "version": "0.6.0",
-  "description": "Orbit browser extension",
+  "description": "The Orbit extension brings all your community data right at your fingertips",
   "repository": "git@github.com:orbit-love/orbit-browser-extension.git",
-  "author": "Nicolas Goutay <nicolas.goutay@gmail.com>",
+  "contributors": [
+    {
+      "name": "Nicolas Goutay",
+      "email": "nicolas.goutay@gmail.com"
+    },
+    {
+      "name": "delete-44",
```

Comment:
Omg I love that guy

Reply:
he’s great i should introduce you someday you’ll have a blast